### PR TITLE
Mock event options to createDynamoDBEvent

### PR DESCRIPTION
**Pull request by Jbarget**
- adds ability to pass options object to createDynamoDBEvent method
- adds tests for above functionality and other mock event methods
- updates readme for documentation on mock event methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,52 +78,52 @@ test('LambdaTest', function(t){
 ```
 
 ### Mock Events
-
 This helper can be used to mock event objects created by AWS services like S3, SNS, or DynamoDB.
 
-The following example uses the `mockEventCreator` to create a mock DynamoDB event.
+```js
+var utils = require('aws-lambda-test-utils');
```
Comment (nelsonic) [inline README.md:84]: thanks for adding the usage example. 👍

File: test/mockEvent.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var test = require('tape');
+
+var mockEventCreator = require('../mockEvent.js');
+
+
+test('mockEventCreator - createDynamoDBEvent', function(t) {
+  t.test('no options: returns an object with 1 dynamo "INSERT" event, with the default eventSourceARN and an awsRegion of "eu-west-1"', function(st) {
```
Comment (nelsonic) [inline test/mockEvent.test.js:9]: not a _huge_ fan of nested tests but see how they are useful for organising here. 👍

**Review by nelsonic — APPROVED** [review]
@Jbarget great work tidying this up! 👍

**Comment by nelsonic** [comment]
@Jbarget `aws-lambda-test-utils@1.1.0` on NPM thanks to you! 🎉  ❤️

File: test/mockEvent.test.js (hunk 2 above)
Comment (Jbarget) [reply test/mockEvent.test.js:9]: @nelsonic neither am i. i didnt get it at first. In the end I went with keeping consistency

**Pull request merged**